core/core.go

```diff
@@ -658,7 +662,7 @@ func (app *App) Start(ctx context.Context) error {
 		defer func() {
 			log.Debug("closing ordersync service")
 		}()
-		if err := app.ordersyncService.GetOrders(innerCtx, ordersyncMinPeers); err != nil {
+		if err := app.ordersyncService.PeriodicallyGetOrders(innerCtx, ordersyncMinPeers, ordersyncMinDelay); err != nil {
```

Comment:
How would you feel about including a log like "started the ordersync service" here? This would make this more consistent with the other goroutines that are created in this function.

Reply:
Good idea. Just added.